site/docs/v1/tech/identity-providers/apple.adoc

```diff
@@ -192,21 +202,39 @@ include::docs/v1/tech/identity-providers/_login-api-integration.adoc[]
 
 == Supporting Web and Mobile
 
-If you are supporting Sign in with Apple from both web and mobile applications, two configurations are required, with slightly different values.
+[NOTE.since]
+====
+Starting with version 1.43.0, FusionAuth supports web and mobile Sign in with Apple using a single IdP configuration.
+====
+
+If you are using version 1.43.0 or later, follow the directions in <<Using Bundle Id>>. For versions prior to 1.43.0, follow the directions in <<Using Services Id with Application Overrides>>.
 
-* For web applications, the [field]#Service Id# is the Apple Service Id.
-* For mobile applications, the [field]#Service Id# is the Apple Bundle Id.
+=== Using Bundle Id
+
+Supporting Sign in with Apple from both web and mobile applications in version 1.43.0 or later no longer requires multiple applications.
+
+* [field]#Services Id# is the Apple Services ID used for web applications
+* [field]#Bundle Id# is the Apple Bundle ID used for native applications
+
+Configure both [field]#Services Id# and [field]#Bundle Id# according to your needs. You can still override the configuration for both fields on a per-application basis.
+
+The login API request from a native application needs to include [field]#data.isNativeApp# with a string value of `"true"`. This will use the [field]#Bundle Id# from the Apple IdP configuration rather than [field]#Services Id#.
```

Comment:
```suggestion
The login API request from a native application needs to include [field]#data.isNativeApp# with a string value of `"true"`. This configuration will use the [field]#Bundle Id# from the Apple IdP configuration rather than [field]#Services Id#.
```

Reply:
What if you provide `data.isNativeApp` of `true` but don't have a configured bundle id. Do we handle that scenario?

Reply:
That scenario returns an `[AppleConfigurationError]` validation message.

Reply:
Sounds good!